source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,273 @@
+
+* `success` - Payment successfully completed
+
+* `failed` - User attempted to make a payment but the payment did not complete
+
+* `cancelled` - Your service cancelled the payment using an API call or the self-service site
+
+* `cancelled - user not eligible` - the user indicated that they are not the sole signatory on the bank account, so the mandate setup could not be completed
+
+* `error` - Something went wrong with GOV.UK Pay or the underlying Payment Service Provider
+
+`cancelled - user not eligible` is a new terminal state related to agreements,
+
+The agreement can go through the following state transitions:
+
+<table style="width:100%">
+  <tr>
+  <th nowrap>Original state</th>
+  <th nowrap>New state</th>
+  <th>Meaning</th>
+  </tr>
+  <tr>
+  <td nowrap><code> CREATED </code></td>
+  <td nowrap><code> STARTED </code></td> 
+  <td> Payer has visited the `next_url` and landed on the GOV.UK Pay page to enter their payment details. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> STARTED </code></td>
+  <td nowrap><code> PENDING </code></td> 
+  <td> Payer has provided their payment information, and confirms their consent to the agreement. <code>PENDING</code> is only used with delayed payment types such as Direct Debit. This indicates that payer consent has been given but the agreement has not yet been confirmed with the issuing bank. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> STARTED </code></td>
+  <td nowrap><code> CANCELLED - USER NOT ELIGIBLE </code></td> 
+  <td> Payer has indicated that they are not the sole signatory for their bank account, and selects the link to go back and try a different payment method. </td>
+  </tr>
+  <tr> 
+  <td nowrap><code> PENDING </code></td>
+  <td nowrap><code> ACTIVE </code></td> 
+  <td> The issuing bank has confirmed that funding source (for example, Direct Debit mandate) is valid. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> PENDING </code></td>
+  <td nowrap><code> FAILED </code></td> 
+  <td> The issuing bank has rejected the funding source. Possible reasons: incorrect bank account details, bank account requires multiple signatories to set up a Direct Debit. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> PENDING </code></td>
+  <td nowrap><code> CANCELLED </code></td> 
+  <td> Cancelled by the payer. </td>
+  </tr>
+  <tr> 
+  <td nowrap><code> ACTIVE </code></td>
+  <td nowrap><code> CANCELLED </code></td>
+  <td> Cancelled by the payer. </td>
+  </tr>
+  </table>
+
+## Payment statuses for Direct Debit
+
+1. `started` - Your service sent an API request to Pay.
+
+2. `pending` - Indicates that payment details have been submitted to the
+payment provider but they are not yet able to confirm whether the payment has
+been successful.
+
+3. The payment reaches one of several finished states:
+
+  * `success` - Mandate set up successfully
+
+  * `failed` - Mandate could not be set up
+
+  * `cancelled` - The user or service cancelled the mandate
```

Comment:
payment was cancelled (need to doublecheck this)

Reply:
Have changed mandate-> payment, will leave this comment thread in until you've double-checked